pytorch_blade/ci_build/build_pytorch_blade.sh

```diff
@@ -43,7 +43,6 @@ function ci_build() {
     else
       extra_args=""
       python3 ../scripts/python/common_setup.py
-      bazel clean --expunge
```

Comment:
Actually don't need to clean the bazel cache ?

Reply:
I had moved the clean to another place.